app/views/feed/_feed_items.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="items mt-0">
+  <% if @revisions.any? %>
+    <% @revisions.each do |revision| %>
+      <% if !@marker_shown && @feed_last_visited_at && revision.created_at < @feed_last_visited_at %>
+        <% @marker_shown = true %>
+
+        <h2 class="feed__heading">Already seen updates</h2>
```

Comment:
```suggestion
        <h2 class="feed__heading">Seen updates</h2>
```
Honestly could take or leave this wording update, but I like the shorter wording.